packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -35,6 +35,11 @@ import {
 import {useSpinButton} from '@react-aria/spinbutton';
 import {useTextField} from '@react-aria/textfield';
 
+interface NumberFieldProps extends AriaNumberFieldProps {
+  /** A ref for the input element. */
+  inputRef: RefObject<HTMLInputElement>
+}
+
```

Comment:
Not 100% sure about this rebase change that I made, perhaps the `inputRef` should be in AriaNumberFieldProps definition. This follows what was last on the branch

Reply:
Nvm, this should be removed. Fixed.